src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -68,9 +68,9 @@ private with sharing class ADDR_Addresses_TEST {
     private static ID adminRecTypeID;
     
     public static void recTypesSetup() {
-        orgRecTypeID = Schema.Sobjecttype.Account.getRecordTypeInfosByName().get('Business Organization').getRecordTypeId();
-        householdRecTypeID = Schema.Sobjecttype.Account.getRecordTypeInfosByName().get('Household Account').getRecordTypeId();
-        adminRecTypeID = Schema.Sobjecttype.Account.getRecordTypeInfosByName().get('Administrative').getRecordTypeId();
+        orgRecTypeID = UTIL_Describe.getBizAccRecTypeID();
```

Comment:
We provide a number of account record types, including one for business organizations, in an unmanaged bundle that the installed puts in before the package.